### PR TITLE
feat(aapd-1226): add appeal details to IP view

### PR DESCRIPTION
**Pull request by Si3rr4wow**
## Ticket Number

https://pins-ds.atlassian.net/browse/AAPD-

## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/Planning-Inspectorate/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by beejjacobs — DISMISSED** [review]
If its not too much work I think a simple version of that would be good... will just reduce that bit of boilerplate of heading, link, etc..

Happy for you to make the call based on ticket size + effort!

File: packages/web-comment/src/routes/appeals/_appealNumber/controller.js [hunk 1]
```diff
@@ -10,7 +25,14 @@ const selectedAppeal = async (req, res) => {
...
 
 	res.render(`appeals/_appealNumber/index`, {
-		appeal: { ...appeal, status: getAppealStatus(appeal) },
+		appeal: {
+			...appeal,
+			status: getAppealStatus(appeal),
+			sections: formatSections({
+				caseData: appeal,
+				sections: formatSections({ caseData: appeal, sections })
+			})
```
[the remark this answers is not in the document]
Comment (Si3rr4wow) [reply packages/web-comment/src/routes/appeals/_appealNumber/controller.js]: no it wasn't 😵‍💫

File: packages/web-comment/src/routes/appeals/_appealNumber/index.njk [hunk 2]
```diff
@@ -1,33 +1,40 @@
...
+        rows: headlineData
+      }) }}
+
+      {% if appeal.casePublished %}
+        <h2 class="govuk-heading-l">Appeal details</h2>
+
+        <p class="govuk-body">
+          <a href="/appeal-details">Appeal details</a>
+        </p>
+      {% endif %}
```
[the remark this answers is not in the document]
Comment (Si3rr4wow) [reply packages/web-comment/src/routes/appeals/_appealNumber/index.njk]: yup, forgot to actually change the template 😅 fixed now

**Review by beejjacobs — COMMENTED** [review]
LGTM!

**Review by beejjacobs — APPROVED** [review]

**Pull request merged**